proto/storage/alert.proto

```diff
@@ -34,9 +34,14 @@ message Alert {
     // Represents an alert on a kubernetes resource (configmaps, secrets, etc.)
     message Resource {
         enum ResourceType {
-            UNKNOWN     = 0;
-            SECRETS     = 1;
-            CONFIGMAPS  = 2;
+            UNKNOWN                    = 0;
+            SECRETS                    = 1;
+            CONFIGMAPS                 = 2;
+            CLUSTERROLES               = 3;
+            CLUSTERROLEBINDINGS        = 4;
+            NETWORKPOLICIES            = 5;
+            SECURITYCONTEXTCONSTRAINTS = 6;
+            EGRESSFIREWALLS            = 7;
```

Comment:
```suggestion
            CLUSTER_ROLES               = 3;
            CLUSTER_ROLE_BINDINGS        = 4;
            NETWORK_POLICIES            = 5;
            SECURITY_CONTEXT_CONSTRAINTS = 6;
            EGRESS_FIREWALLS            = 7;
```

Reply:
Changing the above would be a breaking change to API response, or do I misunderstand?

Reply:
These are new items so changing to underscored would be ok. Still need to test backwards compatibility though

Reply:
Done